did-utils/src/methods/did_peer/method.rs

```diff
@@ -474,24 +475,21 @@ impl DidPeer {
     /// See https://identity.foundation/peer-did-method-spec/#resolving-a-did
     pub fn expand_did_peer_4(&self, did: &str) -> Result<DIDDocument, DIDPeerMethodError> {
         // Ensure long format by computing did:peer:4 short form alias
-
         let alias = Self::shorten_did_peer_4(did)?;
```

Comment:
Please can you improve this comment to make it clearer that hash verification will further not be necessary?